test/components/rankers/test_transformers_similarity.py

```diff
@@ -336,6 +336,7 @@ def __init__(self):
         assert ranker.device == ComponentDevice.from_multiple(DeviceMap.from_hf({"layer_1": 1, "classifier": "cpu"}))
 
     @pytest.mark.integration
+    @pytest.mark.slow
```

Comment:
Same comment here on reducing number of parametrized tests to one.

Reply:
done in https://github.com/deepset-ai/haystack/pull/9296/commits/a31d1574eeede2ea202b44b66e65f09af454e4f9